provider/docker/volume.go

```diff
@@ -206,14 +206,14 @@ func (p *Provider) ReadFile(ctx context.Context, volumeName, relPath string) ([]
 
 	logger.Debug("created getfile container", zap.String("id", cc.ID))
 
-	//defer func() {
-	//	if err := p.dockerClient.ContainerRemove(ctx, cc.ID, types.ContainerRemoveOptions{
-	//		Force: true,
-	//	}); err != nil {
-	//      logger.Error("failed cleaning up the getfile container", zap.Error(err))
-	//		// todo fix logging
-	//	}
-	//}()
+	defer func() {
+		if err := p.dockerClient.ContainerRemove(ctx, cc.ID, types.ContainerRemoveOptions{
+			Force: true,
+		}); err != nil {
+			logger.Error("failed cleaning up the getfile container", zap.Error(err))
+			// todo fix logging
```

Comment:
do we still need this todo?

Reply:
I hope not? Logging works for me just fine.